scripts/local/flake8_plugin.py

```diff
@@ -0,0 +1,177 @@
+import ast
+from collections import namedtuple
+from functools import partial
+
+import pycodestyle
+
+__version__ = "2.0.0"
+
```

Comment:
I'll be cleaning this up and fixing it up in a follow up -- just wanted to copy this over ~as is (pyupgrade fixed a few things already)